packages/gatsby/src/schema/__tests__/queries.js

```diff
@@ -1229,45 +1229,19 @@ describe(`Query schema`, () => {
         `)
       })
 
-      it(`works correctly if GC happen mid running`, async () => {
-        // borrowed from https://unpkg.com/browse/expose-gc@1.0.0/function.js
-        const v8 = require(`v8`)
-        const vm = require(`vm`)
-        v8.setFlagsFromString(`--expose_gc`)
-        const gc = vm.runInNewContext(`gc`)
-
+      it(`works correctly during query execution`, async () => {
         const { clearKeptObjects } = require(`lmdb`)
 
         orderBySpy.mockImplementationOnce((...args) => {
-          // eslint thinks that WeakRef is not defined for some reason :shrug:
```

Comment:
that might be important

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakRef#notes_on_weakrefs